deploy/chart/templates/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -43,29 +69,30 @@ spec:
           - --writePackageServerStatusName
           - {{ .Values.writePackageServerStatusName }}
           {{- end }}
-          {{- if .Values.olm.tlsCertPath }}
+         {{- if .Values.olm.tlsSecret }}
           - --tls-cert
-          - {{ .Values.olm.tlsCertPath }}
-          {{- end }}
-          {{- if .Values.olm.tlsKeyPath }}
+          - /srv-cert/tls.crt
           - --tls-key
-          - {{ .Values.olm.tlsKeyPath }}
+          - /srv-cert/tls.key
+          {{- end }}
+          {{- if .Values.olm.clientCASecret }}
+          - --client-ca
+          - /profile-collector-cert/tls.crt
           {{- end }}
           image: {{ .Values.olm.image.ref }}
           imagePullPolicy: {{ .Values.olm.image.pullPolicy }}
           ports:
             - containerPort: {{ .Values.olm.service.internalPort }}
-            - containerPort: 8081
-              name: metrics
-              protocol: TCP
```

Comment:
Removed given that this should always be 8080 or 8081, as set in the values.yaml